mission/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,15 @@
+<font color='#D81717'>ENEMY FORCES:</font>
+INSERT ENEMIES HERE
+INSERT ENEMY AMLCOA (Most Likely Course of Action, AKA, what they're doing, how they act, how they'll act towards us, what we know, etc)
+
+<font color='#1D69F6'>FRIENDLY FORCES:</font>
+Synixe Contractors, INSERT FRIENDLIES HERE
+
+<font color='#139120'>INDEPENDENT FORCES:</font>
+INSERT INDEPENDENT FORCES IF THERE ARE ANY, OTHERWISE REMOVE THIS LINE AND ONE ABOVE
+
+<font color='#993399'>CIVILIAN CONSIDERATIONS:</font>
+INSERT THINGS TO CONSIDER ABOUT CIVILIANS, PRESENCE/ABSENCE AND/OR BEHAVIOUR/SUPPORT TO FRIENDLIES OR ENEMIES
+
+<font color='#663300'>TERRAIN CONSIDERATIONS:</font>
+INSERT ANYTHING YOU FIND RELEVANT ABOUT HOW THE TERRAIN MAY BE ADVANTAGEOUS OR DISADVANTAGEOUS
```

Comment:
```suggestion
<font color='#E0A750'>SITUATION</font>
INSERT THINGS PLAYERS SHOULD KNOW ABOUT WHERE THEY ARE DEPLOYING. EVENTS THAT HAPPENED LEADING UP TO THEIR DEPLOYMENT OR TO THE GIVEN CONFLICT.

<font color='#D81717'>ENEMY FORCES</font>
INSERT ENEMIES HERE
INSERT ENEMY AMLCOA (Most Likely Course of Action, AKA, what they're doing, how they act, how they'll act towards us, what we know, etc)

<font color='#1D69F6'>FRIENDLY FORCES</font>
INSERT FRIENDLIES HERE

<font color='#139120'>INDEPENDENT FORCES</font>
INSERT INDEPENDENT FORCES IF THERE ARE ANY, OTHERWISE REMOVE THIS LINE AND ONE ABOVE

<font color='#993399'>CIVILIAN CONSIDERATIONS</font>
INSERT THINGS TO CONSIDER ABOUT CIVILIANS, PRESENCE/ABSENCE AND/OR BEHAVIOUR/SUPPORT TO FRIENDLIES OR ENEMIES

<font color='#663300'>TERRAIN CONSIDERATIONS</font>
INSERT ANYTHING YOU FIND RELEVANT ABOUT HOW THE TERRAIN MAY BE ADVANTAGEOUS OR DISADVANTAGEOUS
```

Reply:
remove period after conflict, inconsistent